test-environment/environment.js

```diff
@@ -6,6 +6,27 @@ class CustomEnvironment extends PuppeteerEnvironment {
     async setup() {
         await super.setup();
         this.global.page.setDefaultTimeout(config.defaultTimeout);
+        const chalk = require("chalk");
```

Comment:
enableConsoleLogs() should be used here, instead of code duplication. Page parameter, can be added to method, so it could be reused.